### PR TITLE
Fastroping - Fix equipping FRIES during vehicle init

### PR DESCRIPTION
**Pull request by LinkIsGrim**
**When merged this pull request will:**
- Title.

### IMPORTANT

- [ ] If the contribution affects [the documentation](https://github.com/acemod/ACE3/tree/master/docs), please include your changes in this pull request so the documentation will appear on the [website](https://ace3.acemod.org/).
- [x] [Development Guidelines](https://ace3.acemod.org/wiki/development/) are read, understood and applied.
- [x] Title of this PR uses our standard template `Component - Add|Fix|Improve|Change|Make|Remove {changes}`.

### CODE REVIEW

File: addons/fastroping/functions/fnc_equipFRIES.sqf [hunk 1]
```diff
@@ -25,5 +25,6 @@ if !(isNumber (_config >> QGVAR(enabled))) then {
         private _fries = (getText (_config >> QGVAR(friesType))) createVehicle [0, 0, 0];
         _fries attachTo [_vehicle, getArray (_config >> QGVAR(friesAttachmentPoint))];
         _vehicle setVariable [QGVAR(FRIES), _fries, true];
+        _vehicle addEventHandler ["Deleted", LINKFUNCK(unequipFRIES)];
```
Comment (LinkIsGrim) [inline addons/fastroping/functions/fnc_equipFRIES.sqf:28]:
```suggestion
        _vehicle addEventHandler ["Deleted", LINKFUNC(unequipFRIES)];
```

**Comment by BaerMitUmlaut** [comment]
https://github.com/acemod/ACE3/blob/master/addons/fastroping/XEH_preInit.sqf#L12

**Comment by LinkIsGrim** [comment]
> https://github.com/acemod/ACE3/blob/master/addons/fastroping/XEH_preInit.sqf#L12

I will concede I hadn't seen that, but, it doesn't seem to work.

**Comment by PabstMirror** [comment]
I took a look and it seemed like the original EH was firing
can we get any reproduction steps for it not working?

**Comment by LinkIsGrim** [comment]
> I took a look and it seemed like the original EH was firing can we get any reproduction steps for it not working?

1. Dedicated MP, create FRIES-supported vehicle with createVehicle on server. 
2. On vehicle init, call fnc_equipFRIES.
3. Destroy vehicle via Zeus (END key), then delete vehicle.
4. FRIES object still exists.

**Comment by PabstMirror** [comment]
semi related to #9319
it was running during "init", not "initpost"
global setVars don't work during "init"

**Review by PabstMirror — APPROVED** [review]

**Pull request merged**